### PR TITLE
Remove `cgi.fix_pathinfo=0` as this is not default behaviour

### PR DESCRIPTION
**Pull request by micxer**
It took me hours to figure this one out. There is no way to run *FPM* with *lighttpd* when `cgi.fix_pathinfo` is `0`. Also, by default `cgi.fix_pathinfo` is set to `1`. I suggest to comment this statement and go with the default.

### CODE REVIEW

**Comment by drybjed** [comment]
The setting `cgi.fix_pathinfo=0` was chosen in the past to provide stronger security in `php5-fpm` used with `nginx` (see [this](https://blog.martinfjordvald.com/2011/06/why-path-info-is-the-worst-php-feature-since-register-globals/) and [this](http://www.sitepoint.com/setting-up-php-behind-nginx-with-fastcgi/) and [this](https://github.com/jyr/MNPP/issues/17) and [this](https://bugs.php.net/bug.php?id=68053) and maybe [this](https://serverfault.com/questions/627903/is-the-php-option-cgi-fix-pathinfo-really-dangerous-with-nginx-php-fpm)). I'm not sure yet if reverting that variable to the default is OK now and won't make issues with other applications.

However, it seems that the value might need to be set to `1` for other webservers, [like apache2](https://stackoverflow.com/questions/27207814/why-does-apache2php-fpm-require-cgi-fix-pathinfo-1-and-nginxphp-fpm-doesnt) so maybe lighttpd as well. So I'm willing to go with adding a variable, say `php5_cgi_fix_pathinfo`, default `0` so that this can be configured if needed.

**Comment by micxer** [comment]
Alright, I never had a problem with nginx and fpm. Let me add an option for it and update the PR.

File: templates/etc/php5/fpm/php.ini.j2 [hunk 1]
```diff
@@ -773,17 +773,16 @@ enable_dl = Off
 ; will look for to know it is OK to continue execution.  Setting this variable MAY
 ; cause security issues, KNOW WHAT YOU ARE DOING FIRST.
 ; http://php.net/cgi.redirect-status-env
-;cgi.redirect_status_env = ; #}
+;cgi.redirect_status_env = ;
 
-{#
-; cgi.fix_pathinfo provides *real* PATH_INFO/PATH_TRANSLATED support for CGI.  PHP's
+ cgi.fix_pathinfo provides *real* PATH_INFO/PATH_TRANSLATED support for CGI.  PHP's
```
Comment (drybjed) [inline templates/etc/php5/fpm/php.ini.j2]: This should be commented out.

Comment (micxer) [inline templates/etc/php5/fpm/php.ini.j2]: True. :-/

**Pull request merged**